README.md

```diff
@@ -11,6 +11,103 @@ go get github.com/moolitayer/golang/glog # Dependency
 go get github.com/moolitayer/awx-client-go/awx
 ```
 
+## Usage
+### import
+```
+import 	"github.com/moolitayer/awx-client-go/awx"
+```
+
+### Creating a connection:
+```
+# Uses the builder pattern:
+connection, err := awx.NewConnectionBuilder().
+  Url("http://awx.example.com/api").
```

Comment:
We should probably rename `Url` to `URL`, to stick to the Go naming conventions, but not in this patch.

Reply:
Thanks created https://github.com/moolitayer/awx-client-go/issues/22 so I won't forget